contrib/babelfishpg_tsql/src/catalog.c

```diff
@@ -3250,3 +3250,76 @@ alter_guest_schema_for_db (const char *dbname)
 	table_endscan(tblscan);
 	table_close(bbf_authid_user_ext_rel, RowExclusiveLock);
 }
+
+/*
+ * Update the owner of a database in the catalog.
+ */
+void
+update_db_owner(const char *new_owner_name, const char *db_name)
+{
+	volatile 		Relation sysdatabases_rel;
+	TupleDesc		sysdatabases_rel_descr;
+	ScanKeyData		key;
+	HeapTuple		tuple, db_found;
+	TableScanDesc	tblscan;
+		
+	Datum		values[SYSDATABASES_NUM_COLS];
+	bool		nulls[SYSDATABASES_NUM_COLS];
+	bool		replaces[SYSDATABASES_NUM_COLS];
+
+	/* Do not allow changes to system databases. */
+	/* Note: SQL Server allows changing ownership of msdb. */
```

Comment:
I think we should use T-SQL, not SQL Server here.

Reply:
done